src/main/java/com/univ/lille/copainsderoute/platine/service/EventService.java

```diff
@@ -128,11 +156,69 @@ public void deleteEvent(int id) {
             eventRepository.deleteById(id);
         }
 
-        public Event getEvent(int id) throws RuntimeException {
+        public EventResponseDTOs getEvent(int id) throws RuntimeException {
             Optional<Event> evt = eventRepository.findById(id);
             if (evt == null) {
```

Comment:
```suggestion
if(evt.isPresent())
```
Un optional sera jamais nul, c'est fait pour ça